src/main/java/seedu/address/model/Model.java

```diff
@@ -115,4 +115,13 @@
      * @return True if Liberry is currently in Serve mode.
      */
     boolean isServeMode();
+
+    boolean hasBorrower(Borrower borrower);
+    /**
+     * Returns the user prefs' catalog file path.
+     */
```

Comment:
Should remove irrelevant comments

Reply:
Thanks for spotting the mistakes! Will make a change.